### PR TITLE
V5 ga 24 snackbar

### PR DESCRIPTION
**Pull request by SrdjanReblaze**
https://reblaze.atlassian.net/jira/software/projects/V5GA/boards/39?selectedIssue=V5GA-24

### CODE REVIEW

File: src/components/MessagesSnackBar.vue [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<template>
+  <div v-if="isActive" class="snackbar-holder">
```
Comment (Aviv-Galmidi) [inline src/components/MessagesSnackBar.vue:2]: Please separate each attributes by a new line

File: src/components/MessagesSnackBar.vue [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    <svg-icon v-if="variant=='warning'" width="20" type="mdi" :path="mdiAlert" :class="`${variant} variant-icon`" />
```
Comment (Aviv-Galmidi) [inline src/components/MessagesSnackBar.vue:3]: Please separate each attributes by a new line

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]

File: src/components/MessagesSnackBar.vue (hunk 1 above)
Comment (SrdjanReblaze) [reply src/components/MessagesSnackBar.vue:2]: Ok, sorry for code styling issues, what you are using as code formatter? I am using prettier with some custom settings to make it more similar to project code style

File: src/components/MessagesSnackBar.vue [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+       class="snackbar-holder">
+    <svg-icon v-if="variant=='warning'"
+              width="20"
+              type="mdi"
+              :path="mdiAlert"
+              :class="`${variant} variant-icon`"/>
+    <svg-icon v-if="variant=='error'"
+              width="20"
+              type="mdi"
+              :path="mdiAlert"
```
Comment (Aviv-Galmidi) [inline src/components/MessagesSnackBar.vue:12]: Should this be a different icon? You set it to be the same one as the warning above

Comment (Aviv-Galmidi) [reply src/components/MessagesSnackBar.vue:12]: Implementation

Screen Shot 2023-03-15 at 15 59 10


Design

Screen Shot 2023-03-15 at 15 59 16

Comment (SrdjanReblaze) [reply src/components/MessagesSnackBar.vue:12]: Yes, i didn't find like it is on design on https://pictogrammers.com/library/mdi/

some suggestions?

**Review by Aviv-Galmidi — DISMISSED** [review]

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**